mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length;
```

Comment:
great work, but try to add comments

Reply:
that's a good advice, cheers. will do once i open my laptop again.. I'm on break atm 😹.